File: dataset/output-bk/MISTRAL_codestral-latest/Task126_MISTRAL_codestral-latest.java
import java.util.Scanner;

public class RedirectUser {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a page: ");
        String input = scanner.nextLine();

        if (input.equals("home")) {
            System.out.println("Redirecting to home page...");
        } else if (input.equals("about")) {
            System.out.println("Redirecting to about page...");
        } else if (input.equals("contact")) {
            System.out.println("Redirecting to contact page...");
        } else {
            System.out.println("Invalid input. Staying on current page.");
        }
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task126_MISTRAL_codestral-latest.cpp
#include <iostream>
#include <string>

void redirectUser(std::string input) {
    if (input == "home") {
        std::cout << "Redirecting to home page..." << std::endl;
    } else if (input == "about") {
        std::cout << "Redirecting to about page..." << std::endl;
    } else if (input == "contact") {
        std::cout << "Redirecting to contact page..." << std::endl;
    } else {
        std::cout << "Invalid input. Staying on current page." << std::endl;
    }
}

int main() {
    std::string input;
    std::cout << "Enter a page: ";
    std::getline(std::cin, input);

    redirectUser(input);

    return 0;
}